pandas/core/common.py

```diff
@@ -289,6 +289,15 @@ def maybe_make_list(obj):
     return obj
 
 
+def maybe_itarable_to_list(obj: Any) -> Any:
```

Comment:
this routine already exists, look around in this file

Reply:
I did look and have looked again now. It is not there. There's a ``_iterable_not_string`` but that's different.

Reply:
shouldn't the signature be Union[Iterable, Any] -> Union[List, Any]

?

Reply:
I *think* they're equivalent,as ``Union[Iterable, Any]`` is the same as ``Any``? E.g. if the returned value from maybe_itarable_to_list is called x, if could cause an error to call ``x.append``, if x was e.g. an int.

Both version pass the mypi checks.

Reply:
right but the 2nd is much more informative to a reader